README.md

```diff
@@ -123,7 +123,7 @@ After that, you can install the package in development mode
 
 To compile the Java classes (at the beginning or after changes), run
 
-    python setup.py java
+    python setup.py build_ext
```

Comment:
I think it could be useful to keep this command in the docs for development purposes, as there are situations where a developer may only want to recompile the Java classes - I do think that the preceding line could be edited to make that more clear though